src/main/java/StringCalculator.java

```diff
@@ -0,0 +1,50 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    public static int splitAndSum(String text) {
+        if (text == null) {
+            return 0;
+        }
+        if (text.isEmpty()) {
+            return 0;
+        }
```

Comment:
아래와 같이 표현할 수도 있을 것 같네요~ :) 
```suggestion
private static boolean isBlank(String text) {
     return text == null || text.isEmpty();
}
```